### ISSUE TITLE
Authy import not working

### ISSUE THREAD

**Issue by kohlu** [beemdevelopment/Aegis#265]
###### Info

* __Version__: 1.0.3
* __Source__: Google Play
* __Vault encrypted:__ Yes
* __Device:__ Moto X 2014 (XT1095)
* __Android version and ROM:__ 9, Lineage OS

###### Steps to reproduce

Phone is rooted. 

1. Install the latest version of Authy (24.2.0) from Google Play.
2. Aegis > Settings > Import from app > Authy. (Allow root)
3. Nothing gets imported and error occurs:

```
org.json.JSONException: No value for originalName
org.json.JSONException: No value for decryptedSecret
< decrptedSecret message repeats a dozen more times >
```
Importing the file `com.authy.storage.tokens.authenticator.xml` also shows the same error. Looking into the .xml file, I noticed there are no `decryptedSecret` strings. Instead, I see a bunch of `encryptedSecret` strings. Could be related?

Thanks

**Comment by alexbakker** [comment]
If you enter your password in Authy to decrypt your tokens, you should be able to import them into Aegis afterwards. I'll see if I can implement the decryption logic in Aegis.

**Comment by alexbakker** [comment]
This was fixed by #305.
